libraries/rodeos/include/b1/rodeos/callbacks/filter.hpp

```diff
@@ -16,10 +16,28 @@ struct filter_callbacks {
       derived().get_filter_callback_state().push_data(data.data(), data.size());
    }
 
+   int32_t clock_gettime(int32_t id, uint64_t* data) {
```

Comment:
This should have a different name, so that a libc wrapper can set errno.  (The eosio-tester intrinsic has the same problem).

Reply:
That'd force me to make changes to CDT

Reply:
Which would break eosio-tester

Reply:
We will need to fix this eventually, but I can live with it for now.